### ISSUE TITLE
strange diverging behavior with --docker-run

### ISSUE THREAD

**Issue by exarkun** [datawire/telepresence#60]
I don't really know what's going on here:

```
$ X="-it --rm leastauthority/subscription-converger /app/env/bin/python -c xxx"
$ docker run $X
Traceback (most recent call last):
  File "<string>", line 1, in <module>
NameError: name 'xxx' is not defined
$ ../telepresence --deployment s4-infrastructure --docker-run $X
Starting proxy...
Starting local container...
ImportError: No module named site
Shutting proxy down...
$ 
```

**Comment by itamarst** [comment]
At a guess: `telepresence` adds all the extra environment variables from the pod... perhaps one of them is doing bad things to your program. Probably need some way to distinguish relevant env variables from noise, e.g. looking at which are defined in the Deployment object.

**Comment by itamarst** [comment]
I'm going to change Telepresence so it only copies environment variables explicitly set in the Deployment.

**Comment by itamarst** [comment]
The hypothesized cause is fixed in 0.20. Can you test it actually fixes things for you once it's released? (Later today or tomorrow.) Thanks!
